x-pack/legacy/plugins/canvas/canvas_plugin_src/uis/arguments/image_upload/index.js

```diff
@@ -107,9 +108,15 @@ class ImageUpload extends React.Component {
 
     let selectedAsset = {};
 
-    const urlTypeOptions = [{ id: 'file', label: 'Import' }, { id: 'link', label: 'Link' }];
+    const urlTypeOptions = [
+      { id: 'file', label: ArgumentStrings.ImageUpload.urlTypes.getFile() },
```

Comment:
nit: this is pretty long... I'd almost like to see a destructuring somewhere.

```
import { ArgumentStrings } from '../../../strings';
const { ImageUpload: strings} = ArgumentStrings;
const { urlTypes } = strings;

{ id: 'file', label: urlTypes.getFile() },

displayName: strings.getDisplayName(),
```
etc